docs/source/blocks/extensions.md

```diff
@@ -160,16 +160,16 @@ The `schemaEnhancer` is a generic extension mechanism provided by
 it for the `variation` extension.
 ```
 
-Volto provides a helper to combine multiple schemaEnhancers into a single
-function, so that multiple schema enhancers can be used. For example:
+Volto provides a helper to combine multiple `schemaEnhancer` into a single
```

Comment:
"objects" might not be the correct word, perhaps "extensions" or "extension mechanisms", following the pattern from the previous note? Or we could get technical and call it "thingies". 😜 

```suggestion
Volto provides a helper to combine multiple `schemaEnhancer` objects into a single
```